CodingChallenges/CC_046_Asteroids/Processing/CC_046_Asteroids/Asteroid.pde

```diff
@@ -0,0 +1,68 @@
+// Daniel Shiffman
+// http://codingtra.in
+// http://patreon.com/codingtrain
+// Code for: https://youtu.be/hacZU523FyM
+// Processing transcription: Chuck England
+
+class Asteroid {
+  PVector pos;
+  float r;
+  PVector vel = PVector.random2D();
+  float total = floor(random(5, 15));
+  List<Float> offset = new ArrayList<Float>();
```

Comment:
Will do.

I'll open the code and take a look in just a bit.

If its a place that does not require dynamic adding, I can just use the float[].
Otherwise I will use a FloatList.

To be honest, I did not know about FloatList. And there is all that clumsy mess about using a List<Float> because a List<float> is not possible, and the boxing going on.